### ISSUE TITLE
Training results problem

### ISSUE THREAD

**Issue by lihao612** [tokkiwa/ImageTextCoding#4]
@tokkiwa Excuse me, is the training code provided by the author consistent with the paper? Why is the loss of the local training model tensor(0.5567) different from that provided by the author tensor(0.9948) when the lambda is 3 ?

**Comment by tokkiwa** [comment]
@lihao612 Sorry, what do you mean by the loss? The training loss is not even included in the paper.

**Comment by lihao612** [comment]
@tokkiwa The loss I mentioned is the checkpoint["loss"] of the checkpoint model.Because the results of the personal training model are very different from the model you provided, I checked the model loss and found that the final converged loss is also very different.So I want to ask if the training code is complete
